### ISSUE TITLE
Dark Mode or CSS fix?

### ISSUE THREAD

**Issue by Fenrir200678** [jtydhr88/sd-canvas-editor#8]
Is there a dark mode for the canvas editor planned or any (easy) way to make the UI actually useable? Unfortunately I barely can read the UI buttons, making this not fun to use. I did a fresh install and restarted A1111, but the canvas editor looks like this.
![image](https://user-images.githubusercontent.com/15671097/232040895-592c0636-deaa-45bf-b975-d2e4f0165eac.png)

**Comment by Fenrir200678** [comment]
Thank you for the quick response and your will to check. :)

**Comment by GivenChief** [comment]
I have just registered to describe a similar problem about the unreadable user interface of the extension. And saw that I'm not the only one with this problem.

**Comment by GivenChief** [comment]
> @GivenChief thanks for bringing it up, so is your issue still under dark mode? Or any other situation?

yes, completely the same issues: dark mode of Chrome, SD by Automatic1111, Win11.

**Comment by Fenrir200678** [comment]
Works fine for me now. Thank you! :)

**Comment by GivenChief** [comment]
> fixed in [1fd624c](https://github.com/jtydhr88/sd-canvas-editor/commit/1fd624c18a40f72618177c8b793e0c47611deddb): ![image](https://user-images.githubusercontent.com/860985/232179792-120ee4da-df3c-4336-819e-ef52609a586d.png) let me know it works for you guys


It works perfectly, thanks!
